modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding plugins to the {cli-manager} with YAML files
+
+You can add a CLI plugin to the {cli-manager} using the Form View or YAML View.
+
+[NOTE]
+====
+If you use the form view, add the information for the CLI plugin into the required fields, which are *Name*, *Platforms*, *Files*, *Image*, *Platform*, *Short description*, and *Version*. Then click *Create* to add the CLI plugin.
+====
+
+.Prerequisites
+
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* The {cli-manager} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name>
+spec:
+  shortDescription: short description of the <plugin_name>
+  description: long description of the <plugin_name>
+  version: <version> <1>
+  caveats: <2>
+  homepage: <plugin_homepage>
+  platforms:
+  - platform: <platform> <3>
+    image: <plugin_image>
+    imagePullSecret: <4>
+    files:
+    - from: <plugin_file_path>
```

Comment:
Can you tell me where that `bin` should go? @kasturinarra

Reply:
It's hard to tell from the test case because the indentation is off a bit https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-75289

But if I'm interpreting it right this could go above `files`. Like below (per the test case the plugin name in the `name` field is the same as this field, but someone else can confirm if that's true or not. Also should add a callout for this one providing details / this information if that's true.

```
    imagePullSecret: <4>
    bin: <plugin_name> <5>
    files:
    - from: <plugin_file_path>
```

I would also consider adding some more callouts for a few of the other replaceable values. Short and long description should be clear enough, but there are probably some details that would be useful for the others. 

Like, which field, bin or name, is what you use to call the plugin once it's installed (like a plugin name of `oc-mirror` would use the command `oc mirror` to call it once it's been installed)? The test case uses oc-mirror for both. And does that field have to start with the "oc-\*" or "kubectl-\*" prefix like we say when writing CLI plugins [1]? If so, we should mention that.

This would all be good info to clarify if @kasturinarra or @ardaguclu could confirm.

[1] https://docs.openshift.com/container-platform/4.17/cli_reference/openshift_cli/extending-cli-plugins.html#cli-writing-plugins_cli-extend-plugins